simulator/func_sim/func_sim.h

```diff
@@ -16,6 +16,7 @@
 
 #include <memory>
 #include <string>
+#include <map>
```

Comment:
Please order includes in alphabetical order.

Reply:
https://github.com/MIPT-ILab/mipt-mips/wiki/Code-style-guidelines#order-of-includes